config/kue.ts

```diff
@@ -46,7 +49,43 @@ async function processJob(jobData: any) {
         timestamp, event, data, job_id: jobData._id
       });
       logger.info(oneLine`
-        [i] POSTED event: ${event} to ${notificationResource} was successfull
+        [i] POSTED event: ${event} to ${notificationResource} was successful
+        for Job: ${jobData._id}
+      `);
+    } else if (type === 'SMS') {
+      logger.info(oneLine`
+        [i] Trying to SMS ${notificationResource} with EVENT: ${event}
+        on Timestamp: ${timestamp} for Job: ${jobData._id}
+      `);
+      const accountSid = twilio.sid;
+      const authToken = twilio.token;
+      const fromPhoneNumber = twilio.number;
+      const client = Twilio(accountSid, authToken);
+      const [err, success] = await to (client.messages.create({
+        body: event,
+        to: notificationResource,
+        from: fromPhoneNumber
+      }));
+      logger.info(oneLine`
+        [i] SMS event: ${event} to ${notificationResource} was successful
+        for Job: ${jobData._id}
+      `);
+    } else if (type === 'EMAIL') {
+      logger.info(oneLine`
+        [i] Trying to EMAIL ${notificationResource} with EVENT: ${event}
+        on Timestamp: ${timestamp} for Job: ${jobData._id}
+      `);
+      sgMail.setApiKey(config.sendgrid_key);
+      const msg = {
+        to: notificationResource,
+        from: 'noreply@thinblock.io',
+        subject: `Notification for ${event}`,
+        text: data,
+        html: `<strong>${data}</strong>`,
+      };
+      sgMail.send(msg);
```

Comment:
@abdullahtariq91 if this returns a Promise, do we need to await?

Reply:
I don't think so, the example use case is [here](https://github.com/sendgrid/sendgrid-nodejs/tree/master/packages/mail). Tested it as well, and worked fine.

Reply:
Its actually a promise, [check here](https://github.com/sendgrid/sendgrid-nodejs/blob/master/packages/mail/USE_CASES.md#handling-successfailureerrors) So i think better to await so we know its a success or failure.

Reply:
Added. 👍